test/test_errors.py

```diff
@@ -0,0 +1,18 @@
+from pymongo.errors import (AutoReconnect,
```

Comment:
After importing standard python libraries but before importing pymongo can you add `sys.path[0:0] = [""]`. For example: https://github.com/mongodb/mongo-python-driver/blob/4760d0781505c0f13d5d1bc061ff89075927eb2f/test/test_bson.py#L27